ooniprobe/Utility/OONIApi.m

```diff
@@ -21,6 +21,12 @@ + (void)checkIn:(NSString*)runType onSuccess: (void (^)(NSArray*))successcb onEr
     if (error != nil) {
         return;
     }
+    
+    if ([SettingsUtility getNumberCategoriesEnabled] <= 0){
+        errorcb(error);
```

Comment:
In this case the error is `nil` because of the previous `if`. I would like to recommend to instantiate a proper error here and then invoke the callback with such an error (this can and should probably be a one-liner).